aten/src/ATen/CMakeLists.txt

```diff
@@ -354,6 +377,16 @@ IF(CUDA_FOUND)
   ENDIF($ENV{ATEN_STATIC_CUDA})
 ENDIF()
 
+### Link in the ROCm stuff ###
+FIND_LIBRARY(HIPBLAS_LIBRARY hipblas HINTS ${HIPBLAS_PATH}/lib)
```

Comment:
This looks like it is being unconditionally searched for. You didn't REQUIRE it, but wouldn't it be safer to only search if WITH_ROCM?